tests/workspace.bzl

```diff
@@ -1,19 +1,24 @@
 load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@rules_scala_annex//rules/scala:workspace.bzl", "zinc_version")
+
+specs2_version = "4.20.9"
+scalatest_version = "3.2.19"
+scalapb_version = "0.11.17"
```

Comment:
Nit: Could we import this from `@rules_scala_annex//rules/scala:workspace.bzl`?

Reply:
Yep. Good catch.